pyproject.toml

```diff
@@ -69,6 +69,7 @@ rank-bm25 = {optional = true, version = "^0.2.2"}
 scikit-learn = {optional = true, version = "<1.3.0"}
 spacy = {optional = true, version = "^3.7.1"}
 aiostream = "^0.5.2"
+voyageai = "^0.1.2"
```

Comment:
Please remove from here in favour of having the user have to install this library as standalone (`pip install voyageai`) only when they actually use this embedding class. See comment above.